packages/sqlgg/sqlgg.20231201/opam

```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+maintainer: "ygrek@autistici.org"
+authors: ["ygrek"]
+homepage: "https://ygrek.org/p/sqlgg/"
+dev-repo: "git+https://github.com/ygrek/sqlgg.git"
+bug-reports: "https://github.com/ygrek/sqlgg/issues"
+license: "GPL-2.0-only"
+tags: [ "org:ygrek" ]
+build: [
+  ["dune" "subst"] {dev}
+  ["dune" "build" "-p" name "-j" jobs "@install" "@doc" {with-doc}]
+]
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {>= "2.0"}
+  "menhir"
+  "mybuild"
+  "ppx_deriving"
```

Comment:
This should help fixing some of the lower bound failures like [lower-bounds (failed: No ppx driver were found. It seems that ppx_deriving.std is not)](https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/d280e6363d8e0147c11e8272c24c4a288704c0ea/variant/compilers,4.07,sqlgg.20231201,lower-bounds)

```suggestion
  "ppx_deriving" {>= "4.3"}
```